servers/rendering/rendering_device_graph.h

```diff
@@ -590,6 +594,7 @@ class RenderingDeviceGraph {
 	static RDD::BarrierAccessBits _usage_to_access_bits(ResourceUsage p_usage);
 	int32_t _add_to_command_list(int32_t p_command_index, int32_t p_list_index);
 	void _add_adjacent_command(int32_t p_previous_command_index, int32_t p_command_index, RecordedCommand *r_command);
+	int32_t _add_to_slice_read_list(int32_t p_command_index, Rect2i suberesources, int32_t p_list_index);
 	int32_t _add_to_write_list(int32_t p_command_index, Rect2i suberesources, int32_t p_list_index);
```

Comment:
```suggestion
	int32_t _add_to_write_list(int32_t p_command_index, Rect2i p_subresources, int32_t p_list_index);
```
Let's also fix this typo